thetis/coordsys.py

```diff
@@ -0,0 +1,97 @@
+"""
+Generic methods for converting data between different spatial coordinate systems.
+Uses pyproj library.
+"""
+
+import pyproj
+import numpy as np
+
+# NOTE this adds pyproj as dependencency
+# TODO generalize; define unique names for commonly-used systems
+
+
+UTM_ZONE10 = pyproj.Proj(
+    proj='utm',
+    zone=10,
+    datum='NAD83',
+    units='m',
+    errcheck=True)
+SPCS_N_OR = pyproj.Proj(init='nad27:3601', errcheck=True)
+LL_WGS84 = pyproj.Proj(proj='latlong', datum='WGS84', errcheck=True)
+LL_WO = pyproj.Proj(proj='latlong', nadgrids='WO', errcheck=True)  # HARN
```

Comment:
What is nadgrids='WO' ? Maybe this is all a little specific?

Reply:
Yes this is not actually needed in any test case, so I've removed it.